test/activate-event.html

```diff
@@ -49,8 +49,13 @@
 
       var s;
 
-      setup(function () {
+      setup(function(done) {
         s = fixture('test');
+
+        s.addEventListener('iron-items-changed', function onIronItemsChanged() {
```

Comment:
This is because of a breaking change that's a combined effort of IronSelectableBehavior, Polymer, and the custom element reaction queues introduced in v1. I left a really vague note about this [here](https://github.com/PolymerElements/iron-selector/pull/139/files#diff-c64eedc3daead72759c9014ab0eb08c4R19). The gist is that IronSelectableBehavior can't update its items and selected states synchronously within `attached` without breaking its most basic use case.

Specifically, if you have an element with IronSelectableBehavior with an `attr-for-selected` value and an item (1) with a property that reflects to that same attribute referenced by `attr-for-selected`, (2) this property has a default value of `false`, and (3) this item will be the selected item; then you end up with this lame series of events:
  1. The tree is cloned / constructed and the default values are stored by Polymer in some internal object.
  1. The tree is connected and `connectedCallback` reactions are enqueued for the selector and the item.
  1. The selector's `connectedCallback` is dequeued and run first, during which it calls `setAttribute` on the item.
  1. The item's `connectedCallback` is dequeued and run *as the next frame on the stack below the function calling `setAttribute`* which then reads the currently known attribute value out of the internal object and reflects it by calling `removeAttribute` because the default is `false` and `attributeChangedCallback` hasn't yet been called to indicate otherwise.
  1. The item's *first* pending `attributeChangedCallback` reaction is called (enqueued by `setAttribute`), setting the property to `true`.
  1. The item's *second* pending `attributeChangedCallback` reaction is called (enqueued by `removeAttribute`), setting the property back to `false`.

So, the selector ends up trying to call `setAttribute` on an item which already has a pending `connectedCallback` and that `connectedCallback` ends up invalidating the `setAttribute` call by reflecting that property's default value back to the attribute (i.e. removing it). This exact scenario is what happens with `paper-radio-group`, `paper-radio-button` and the `checked` attribute.

`forceSynchronousItemUpdate` works here as well but I'd rather encourage people that run across this to use `iron-items-changed`.